.travis.yml

```diff
@@ -7,6 +7,7 @@ install:
   - unzip -j -n /tmp/infra-utils.zip -x "infra-utils-master/.gitignore" "infra-utils-master/LICENSE" "infra-utils-master/*.md" "infra-utils-master/aws/*"
   - ./setup_aws_cli.sh || travis_terminate 1
   - pip install pre-commit aws-sam-cli
+  - pip install -r lambdas/cfn-set-bucket-tags/set_bucket_tags/requirements.txt
```

Comment:
is this so you can run your tests?  it's inappropriate to install that here.

Reply:
In that case we need to discuss how we are going to continue to develop separate python applications with possibly conflicting requirements in the same repo and run their tests.